1.0/docs/devguide/properties.md

```diff
@@ -324,14 +324,69 @@ Example:
 
     });
 
-**Compatibility note:** The argument order for change handlers is currently the
-**opposite** of the order used in 0.5. 
-{: .alert .alert-info }
-
 Property change observation is achieved in Polymer by installing setters on the
 custom element prototype for properties with registered interest (as opposed to
 observation via `Object.observe` or dirty checking, for example).
 
+### Always include dependencies as observer arguments
```

Comment:
This heading (and some of the text below) doesn't make sense for single-property observers, since you can't include any additional dependencies.
 
In the current organization, it feels like this should perhaps be two sections.  Maybe a short warning in one section cross-referencing a more general section.